bodhi/tests/client/test_bindings.py

```diff
@@ -858,6 +858,25 @@ def test_without_type_(self):
         kwargs['csrf_token'] = 'a token'
         client.send_request.assert_called_once_with('updates/', verb='POST', auth=True, data=kwargs)
 
+    def test_with_edited(self):
```

Comment:
This test seems to do the same assertions as  the `test_without_type_`. So I am not sure this is needed.